src/Context/EntityBrowserContext.php

```diff
@@ -0,0 +1,127 @@
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$item_selector\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$item_selector\").click()");
+  }
+
+  /**
+   * @Given Item with label :label in entity browser :entity_browser should have the class :class_name
+   */
+  public function itemInEntityBrowserShouldHaveClass($label, $enity_browser, $class_name) {
+    $item_selector = $this->getEntityBrowserItemSelector($label);
+    $entity_browser_selector = $this->getEntityBrowserSelector($enity_browser);
+    $this->getSession()->getDriver()->wait(1000, "");
+    $result = $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$item_selector\").closest('.views-row').hasClass(\"$class_name\")");
+    if (!$result) {
+      throw new ExpectationException('Class not set.', $this->getSession());
+    }
+  }
+
+  /**
+   * @Then I wait for :element_selector in entity browser :entity_browser
+   */
+  public function iWaitForInEntityBrowser($element_selector, $entity_browser) {
+    $entity_browser_selector = $this->getEntityBrowserSelector($entity_browser);
+    $this->getSession()->getDriver()->wait(5000, "jQuery(\"$entity_browser_selector\").contents().find(\"$element_selector\").length > 0");
+  }
+
+  /**
+   * @Given I click on :element_selector in entity browser :entity_browser
+   */
+  public function iClickOnInEntityBrowser($element_selector, $entity_browser) {
+    $entity_browser_selector = $this->getEntityBrowserSelector($entity_browser);
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$element_selector\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$element_selector\").click()");
+  }
+
+  /**
+   * @Given I wait for entity browser :entity_browser to close
+   */
+  public function iWaitForEntityBrowserToClose($entity_browser) {
+    $entity_browser_selector = $this->getEntityBrowserSelector($entity_browser);
+    $this->getSession()->getDriver()->wait(5000, "jQuery(\"$entity_browser_selector\").length == 0");
+  }
+
+
+
+  /**
+   * @Then :element_selector in entity browser :entity_browser should have the class :class_name
+   */
+  public function inEntityBrowserShouldHaveTheClass($element_selector, $entity_browser, $class_name) {
+    $entity_browser_selector = $this->getEntityBrowserSelector($entity_browser);
+    $result = $this->getSession()
+      ->evaluateScript("jQuery(\"$entity_browser_selector\").contents().find(\"$element_selector\").hasClass(\"$class_name\")");
```

Comment:
hm, not sure it's good to start using so much jquery checks. The checks should work with the mink API as well since it takes the html after javascript is executed. Anyway, since you got it working already, let's keep it like that.